src/services/dashboards/resource.js

```diff
@@ -1496,7 +1929,7 @@ Expected JSON:
             title: 'Jan-22',
              value: '14',
           },
-          {
+          {79
```

Comment:
what is this change?

Reply:
Odd looks like a typo surprised lint didn't catch it.

Reply:
ah its in a comment